pkg/cloudprovider/aws/packing/packing.go

```diff
@@ -77,33 +90,76 @@ func (p *podPacker) Pack(ctx context.Context, constraints *cloudprovider.Constra
 	return packings, nil
 }
 
-// TODO filter instance types based on node contraints like availability zones etc.
-func (p *podPacker) getNodeCapacities(constraints *cloudprovider.Constraints) []*nodeCapacity {
+func (p *podPacker) getNodeCapacities(constraints *cloudprovider.Constraints) ([]*nodeCapacity, error) {
 	result := make([]*nodeCapacity, 0)
```

Comment:
Optional: could you change this to not use `make`? I don't think our house style is to use `make` unless there's a reason (like the size needs to be set in advance).

Reply:
Yeah, I actually had it not using make in the first rev and then changed it when I rebased because I saw that more code in that file was using `make`.  I prefer the `slice{}` syntax anyways, so I'll change these back.

Reply:
Thanks, feel free to convert any `make...0` cases back to slice syntax, I think they're just oversights.